mobile-widgets/qml/main.qml

```diff
@@ -321,21 +321,12 @@ Kirigami.ApplicationWindow {
 						name: ":/icons/cloud_sync.svg"
 					}
 					text: qsTr("Manual sync with cloud")
-					enabled: prefs.credentialStatus === CloudStatus.CS_VERIFIED ||
-							prefs.credentialStatus === CloudStatus.CS_NOCLOUD
+					enabled: PrefCloudStorage.cloud_verification_status === CloudStatus.CS_VERIFIED
 					onTriggered: {
-						if (prefs.credentialStatus === CloudStatus.CS_NOCLOUD) {
-							returnTopPage()
-							prefs.oldStatus = prefs.credentialStatus
-							manager.startPageText = "Enter valid cloud storage credentials"
-							prefs.credentialStatus = CloudStatus.CS_UNKNOWN
-							globalDrawer.close()
```

Comment:
I think you are misunderstanding the intent. The logic was that if you hit "manually sync" and are in no-cloud mode, you are prompted to log in with a cloud account. So you are removing an intentional feature.
I'm not saying you are wrong to remove it, just that the way you describe this is a bit misleading.

Reply:
Yes I really misunderstood that, and as a User I would be confused by getting the login screen without a warning or explanation.

Please feel free to edit the commit messages.